### DISCUSSION TITLE
FZF_DEFAULT_OPTS in conf.d/fzf.fish can't be overwritten

### DISCUSSION THREAD

**Discussion by poperigby** [PatrickF1/fzf.fish#106] [Q&A] [answered]
I followed the suggestion of overwriting `FZF_DEFAULT_OPTS` found in `conf.d/fzf.fish` by doing something like this `set --universal --export FZF_DEFAULT_OPTS --height 50% --margin 1`, but it doesn't apply the settings, and just gives me this warning:

```
set: Universal variable 'FZF_DEFAULT_OPTS' is shadowed by the global variable of the same name.
```

**Comment by PatrickF1** [comment]
It is overshadowed, but if you open a new fish session and `echo $FZF_DEFAULT_OPTS`, you'll see that it is now `--height 50% --margin 1` rather than the defaults set by fzf.fish. Feel free to comment back if it doesn't work for you.

**Comment by poperigby** [comment]
I just tried that and it didn't work.

**Comment by PatrickF1** [comment]
Can you describe exactly what you did? I can't repro your issue

**Comment by PatrickF1** [comment]
Hi @PopeRigby, here's an explanation of the error you got. https://github.com/fish-shell/fish-shell/issues/4267
As it explains, as long as the variable doesn't get set again by a config file, the universal variable you set will be used. fzf.fish's config avoids overriding any value you set for `FZF_DEFAULT_OPTS` by using a guard statement. See https://github.com/PatrickF1/fzf.fish/blob/2cc2479237eb1a0c8a1f127215dd6b97358523c8/conf.d/fzf.fish. Is it possible you have FZF_DEFAULT_OPTS set somewhere else in your config files?

**Comment by poperigby** [comment]
Not that I can think of. Is there a way to see all the places a variable is being set?

**Comment by PatrickF1** [comment]
Can you
1. give me the output of `set --show FZF_DEFAULT_OPTS`
2. try uninstalling and reinstalling this plugin to see if that'll help

**Comment by poperigby** [comment, marked as answer]
There we go, uninstalling and reinstalling fixed it. Thanks!

**Comment by PatrickF1** [comment]
No problem!
